src/xhr.js

```diff
@@ -55,7 +66,11 @@ class Xhr {
    * @return {!Promise<!JSONValue>}
    */
   fetchJson(input, opt_init) {
-    return this.fetch_.call(null, input, opt_init).then(response => {
+    const init = opt_init || {};
```

Comment:
Yeah, defaulting it here is totally fine - as long as we allow optional argument here.